tensorflow/core/profiler/internal/tfprof_stats_test.cc

```diff
@@ -84,12 +84,12 @@ TEST_F(TFProfStatsTest, CustomOpType) {
       "name: \"_TFProfRoot\"\ntotal_exec_micros: 13\ntotal_parameters: "
       "451\nchildren {\n  name: \"DW\"\n  exec_micros: 2\n  parameters: 162\n  "
       "total_exec_micros: 2\n  total_parameters: 162\n  devices: "
-      "\"/job:localhost/replica:0/task:0/gpu:0\"\n  cpu_exec_micros: 2\n  "
+      "\"/job:localhost/replica:0/task:0/device:GPU:0\"\n  cpu_exec_micros: 2\n  "
```

Comment:
these test use data from testdata. So I guess it might still be /gpu:0. It good to keep it pass for backward compatibility